### ISSUE TITLE
gqlgen init on windows

### ISSUE THREAD

**Issue by maeglindeveloper** [99designs/gqlgen#276]
Hi,

I'm actually trying to use `gqlgen init` using a `schema.graphql` file, but it does not seem to generate the appropriate code. 
It seems that the package are not found since the path are not well formated. I'm running it on Windows.

### Expected Behaviour
generate all the files expected by 
https://gqlgen.com/getting-started/

-     gqlgen.yml - The gqlgen config file, knobs for controlling the generated code.
-     generated.go - The graphql execution runtime, the bulk of the generated code
-     models_gen.go - Generated models required to build the graph. Often you will override these with models you write yourself. Still very useful for input types.
-     resolver.go - This is where your application code lives. generated.go will call into this to get the data the user has requested.

### Actual Behavior

```
exec plan failed: unable to resolve package for D:/dev/go/src/todo.Todo: cannot find package "D:/dev/go/src/todo" in any of:
        C:\Go1.10.3\src\D:\dev\go\src\todo (from $GOROOT)
        D:\dev\go\src\D:\dev\go\src\todo (from $GOPATH)
```
The path is definitively not good. 

### schema.graphql

```
# GraphQL schema example
#
# https://gqlgen.com/getting-started/

type Todo {
  id: ID!
  text: String!
  done: Boolean!
  user: User!
}

type User {
  id: ID!
  name: String!
}

type Query {
  todos: [Todo!]!
}

input NewTodo {
  text: String!
  userId: String!
}

type Mutation {
  createTodo(input: NewTodo!): Todo!
}
```
Thanks in advance!
@vektah

**Comment by vektah** [comment]
Fixed in #312 and tests added in #299

**Comment by maeglindeveloper** [comment]
@vektah Thanks for the fix! :)
